packages/manager/src/managers/prismicRepository/types.ts

```diff
@@ -167,3 +167,32 @@ export const Environment = t.type({
 	),
 });
 export type Environment = t.TypeOf<typeof Environment>;
+
+export const OnboardingStepId = t.union([
+	t.literal("createPrismicProject"),
+	t.literal("chooseLocale"),
+	t.literal("createProject"),
+	t.literal("setupSliceMachine"),
+	t.literal("createPageType"),
+	t.literal("codePage"),
+	t.literal("createSlice"),
+	t.literal("reviewAndPush"),
+	t.literal("createContent"),
+	t.literal("renderPage"),
+]);
+export type OnboardingStepId = t.TypeOf<typeof OnboardingStepId>;
+
+export const OnboardingState = t.type({
```

Comment:
⚠️ For everything relating to types validation, and for new code, we want to use Zod and not io-ts. Only exception i prismic-types-internal as it's whole project. But now for new code where we want to validate a simple thing like that, Zod should be used.

Reply:
The only issue, as of now, is that this will get passed to utils like `decode`, which accept a io-ts type

Reply:
I would take the time to just do the decode with io-ts, as we use zod everywhere else for new code at Prismic, it's not very long to update only for here, no?

Reply:
Done ✅